src/components/bark/bark-button.tsx

```diff
@@ -27,13 +28,18 @@ export function BarkButton(props: {
         className={clsx("h-[60px]", className)}
         variant={variant}
         onClick={onClick}
+        type={type}
       >
         {children}
       </Button>
     );
   }
   return (
-    <Button className={clsx("h-[60px]", className)} variant={variant}>
+    <Button
+      className={clsx("h-[60px]", className)}
+      variant={variant}
+      type={type}
+    >
```

Comment:
Now that this supports submit, we should add a TODO to deprecate BarkFormSubmitButton. The "mt-6" makes it hard to use when it needs to be arranged with other buttons.

Reply:
Deprecated BarkFormSubmitButton 80ce08e